Tools/CISupport/build-webkit-org/factories_unittest.py

```diff
@@ -649,6 +649,16 @@ class TestExpectedBuildSteps(unittest.TestCase):
             'builtins-generator-tests',
             'trigger-crash-log-submission'
         ],
+        'Apple-BigSur-XCBuild-Release-Build': [
+            'configure-build',
+            'configuration',
+            'clean-and-update-working-directory',
+            'show-identifier',
+            'kill-old-processes',
+            'delete-WebKitBuild-directory',
```

Comment:
Could we persist WebKitBuild between builds, like EWS does? One thing I'd like this bot to be testing is whether incremental builds are successful.

Reply:
Though impossible to determine from this unit test, the 'delete-WebKitBuild-directory' step would only run if someone triggered a clean build.

Reply:
Ok, nevermind then :)